### PR TITLE
Improved NTNetRelay Interface

### PR DESCRIPTION
**Pull request by Geevies**
I encountered an issue when I was using the in-game IRC, where I had the "send message" button was above the box where the chat takes place, so when the chat eventually grew to a rather large size, you had to scroll up above the last message to hit "send message", which feels awkward and not very fun, having to scroll up and down the whole time.

So, I separated the chat controls into stuff that doesn't have to do with messages, and stuff that has to do with messages, and placed the one above the box, and the other beneath the box, resulting in this:
image

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: nano/templates/ntnet_chat.tmpl [hunk 1]
```diff
@@ -46,10 +49,13 @@
...
 		</div>
         </div>	
- 	<b>Connected Users</b><br>
-	{{for data.clients}}
-		{{:value.name}}<br>
-	{{/for}}
+	<div class="itemLabel">
+		Messaging: 
+	</div>
+	<div class="itemContent">
```
Comment (Arrow768) [inline nano/templates/ntnet_chat.tmpl]: You are opening a div here, but are not closing it

Comment (Geevies) [reply nano/templates/ntnet_chat.tmpl]: Donezo.

File: nano/templates/ntnet_chat.tmpl [hunk 2]
```diff
@@ -46,10 +49,14 @@
...
+		<table>
+			<tr><td>{{:helper.link("Send message", null, {'PRG_speak' : 1})}}
```
Comment (skull132) [inline nano/templates/ntnet_chat.tmpl]:
```suggestion
			<tr><td>{{:helper.link("Send message", null, {'PRG_speak' : 1})}}</td></tr>
```

Comment (Geevies) [reply nano/templates/ntnet_chat.tmpl]: yes

**Pull request merged**